### PR TITLE
Adding two new properties to the "HandlingData" file.

### PR DESCRIPTION
**Pull request by NiziulLuizin**
+ "PetrolConsumptionRate"
+ "WeaponDamageScaledToVehicleHealthMultiplier"

### CODE REVIEW

File: source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs [hunk 1]
```diff
@@ -738,6 +738,29 @@ public float PetrolTankVolume
 			}
 		}
 
+		public float PetrolConsumptionRate
```
Comment (kagikn) [inline source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs:741]: I couldn't find any occurrences of `fPetrolConsumptionRate` in any handling.meta files, but I can find a occurrence of the hashed value `0x80593873` (by joaat func without applying case conversion) of `fPetrolConsumptionRate` in dumped exe. I can confirm the offset is right after `fOilVolume` too.

File: source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs [hunk 2]
```diff
@@ -1307,6 +1330,29 @@ public float WeaponDamageMultiplier
 			}
 		}
 
+		public float WeaponDamageScaledToVehicleHealthMultiplier
```
Comment (kagikn) [inline source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs:1333]: So this one is for `fWeaponDamageScaledToVehHealthMult`, which is used only for the Rhino and Khanjali in vanilla handling.meta files.

**Review by kagikn — APPROVED** [review]
I don't have any feedbacks that must be addressed in naming convention.

**Comment by crosire** [comment]
Would be nice to have some basic XMLDoc for these, similar to the other properties.

**Pull request merged**